tests/utils_tests/test_datastructures.py

```diff
@@ -48,7 +48,7 @@ def test_multivaluedict(self):
             [('name', ['Adrian', 'Simon']), ('position', ['Developer'])]
         )
 
-        with self.assertRaisesMessage(MultiValueDictKeyError, 'lastname'):
+        with self.assertRaisesMessage(MultiValueDictKeyError, "'lastname'"):
```

Comment:
This doesn't act as a regression test because it tests if `"'lastname'"` is in the exception message which it already is. You want to test if the quotes are removed. This would do it:
```python
with self.assertRaises(MultiValueDictKeyError) as cm:
    d.__getitem__('lastname')
self.assertEqual(str(cm.exception), "'lastname'")
```